apps/editors/tests/test_views.py

```diff
@@ -2046,7 +2046,14 @@ def test_action_links_as_admin(self):
         ]
         check_links(expected, pq(r.content)('#actions-addon a'), verify=False)
 
-    def test_action_links_as_admin_unlisted_addon(self):
+    def test_unlisted_addon_action_links(self):
+        """No "View Listing" link for the standard reviewers."""
```

Comment:
Not a fan of the wording. Maybe something like `No "View Listing" link for unlisted addons, and not action links for standard reviewers`. And then below `No "View Listing" link for unlisted addons, and "Edit"/"Manage" links for admins` ? To make it clear that nobody is going to get `View Listing` link for unlisted Addons anyway, and only admins can see Edit/Manage.